src/index.ts

```diff
@@ -44,7 +61,12 @@ async function run() {
       const cacheDate = new Date(cache.last_accessed_at);
```

Comment:
You are always checking against `last_accessed_at`, shouldn't that be dynamic based on `created` and `accessed`?

Reply:
You just changed the logging. Shouldn't it be something like this?

```js
if(created && accessed) {
  // Check created and/or accessed
} else if (created) {
  // Check created
} else if(accessed) {
  // Check accessed
}
```